apps/server/src/modules/board/controller/card.controller.ts

```diff
@@ -8,12 +8,13 @@ import {
 	CardIdsParams,
 	CardListResponse,
 	CardUrlParams,
+	ElementsTypeParams,
 	MoveCardBodyParams,
 	TextElementResponse,
 } from './dto';
-import { CardResponseMapper, TextElementResponseMapper } from './mapper';
+import { CardResponseMapper, ElementsResponseMapper } from './mapper';
 
-@ApiTags('Cards')
+@ApiTags('Board/Cards')
 @Authenticate('jwt')
 @Controller('cards')
```

Comment:
![image](https://user-images.githubusercontent.com/106820326/230284603-9934ce2c-8df5-466a-8654-3c602b6c5b28.png)
So this comes from milky way? And will later be integrated into the this endoint?

And are we fine with elements, columns, and cards being a top level endpoint? Could there be a naming conflict in the future? I could also think of 'boards/columns' being the endpoint here.

Reply:
Ops is only purely ruscht, will be removed later. It was only so I can distinguish them.

Reply:
/cards/task was created by Team MilkyWay and the idea is to migrate this as well as old tasks, old lessons and the roomsDetail-Page into the board-structure... step by step.

We had elements, cards and columns as endpoints below /boards until this week (e.g. /boards/:boardId/columns/:columnId/cards/...) but decided to change that as information like boardId and columnId are not needed to work with a card...
Furthermore having them as top level endpoints the structure is not fixed with api - so if there is any demand like allowing cards on whiteboards (and not only in columns of columnboards) or elements directly in columns (and not only on cards) - we do not have to change the urls.
But maybe I forgot some arguments for one solution or the other.

Reply:
Yes, we did not want to use the whole path for accessing a board object. We think it is sufficent to just specify the parent object when creating or listing child objects.